runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java

```diff
@@ -239,4 +239,15 @@ default boolean swapDecant()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+			position = 19,
+			keyName = "swapBuyPlank",
+			name = "Buy-plank",
+			description = "Swap Talk-to with Buy-Plank for Sawmill Operator at the Lumber Yard."
+	)
+	default boolean swapBuyPlank() 
+	{ 
+		return false;

```

Comment:
This seems like good default to be true? Not sure, never made planks myself

Reply:
Trade is the default when you have the trade menu entry swapper setting on.